pandas/core/groupby/groupby.py

```diff
@@ -3411,38 +3411,42 @@ def rank(
     @final
     @Substitution(name="groupby")
     @Appender(_common_see_also)
-    def cumprod(self, axis: Axis = 0, *args, **kwargs) -> NDFrameT:
+    def cumprod(self, axis: Axis = 0) -> NDFrameT:
         """
         Cumulative product for each group.
 
         Returns
         -------
         Series or DataFrame
         """
-        nv.validate_groupby_func("cumprod", args, kwargs, ["numeric_only", "skipna"])
+        nv.validate_groupby_func("cumprod", ["numeric_only", "skipna"])
         if axis != 0:
-            f = lambda x: x.cumprod(axis=axis, **kwargs)
+            f = lambda x: x.cumprod(
+                axis=axis,
+            )
             return self._python_apply_general(f, self._selected_obj, is_transform=True)
 
-        return self._cython_transform("cumprod", **kwargs)
+        return self._cython_transform(
+            "cumprod",
+        )
 
     @final
     @Substitution(name="groupby")
     @Appender(_common_see_also)
-    def cumsum(self, axis: Axis = 0, *args, **kwargs) -> NDFrameT:
+    def cumsum(self, axis: Axis = 0) -> NDFrameT:
         """
         Cumulative sum for each group.
 
         Returns
         -------
         Series or DataFrame
         """
-        nv.validate_groupby_func("cumsum", args, kwargs, ["numeric_only", "skipna"])
```

Comment:
The CI is failing because of this. You can't simply delete the values, you should provide an empty tuple and an empty dictionary instead.

Reply:
Yes will do

Reply:
i am getting this error `Error: The runner has received a shutdown signal...' in GHA. GH 45651` , could you help on this.